### PR TITLE
Switch to @loadable/component

### PR DESCRIPTION
**Pull request by fkorotkov**
In regards of discussion in #155

/cc @RDIL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,15 +22,16 @@
     "graphiql": "^0.17.5",
     "graphql": "^14.6.0",
     "graphql-relay": "^0.6.0",
+    "install": "^0.13.0",
     "js-base64": "^2.5.2",
+    "npm": "^6.14.1",
```
Comment (RDIL) [inline package.json]: Why is this being added?

File: package.json [hunk 2]
```diff
@@ -38,6 +40,8 @@
     "recharts": "1.8.5"
   },
   "devDependencies": {
+    "@loadable/babel-plugin": "^5.12.0",
+    "@loadable/webpack-plugin": "^5.12.0",
```
Comment (RDIL) [inline package.json]: This won’t do anything unless it is initialized in a webpack.config.js which would require ejecting

File: package.json (hunk 1 above)
Comment (fkorotkov) [reply package.json]: Probably npm install added it. Will remove

**Comment by fkorotkov** [comment]
https://github.com/cirruslabs/cirrus-ci-web/pull/159 is even better

**Pull request closed without merging**